gameModes.py

```diff
@@ -185,7 +186,10 @@ def __init__(self):
         self.allowConsecutiveHitsBonus = False
         self.allowConsecutiveMissesBonus = False
         self.allowLevelupBonus = False
-        self.name = ALL_MODES_STR[2]
+        if globalVars.stressrelease:
+            self.name = ALL_MODES_STR[4]

```

Comment:
If statement here is bad. Reading existing codebase and keeping the conventions is extremely important.